### PR TITLE
Firebase service tests

### PR DESCRIPTION
**Pull request by amankagarwal**

### CODE REVIEW

File: src/test/java/com/careerfair/q/service/database/implementation/FirebaseServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,535 @@
...
+        } catch (FirebaseException ex) {
+            assertEquals(ex.getMessage(), "No student with email=s2@u2.edu exists");
+        }
+    }
+
+    @Test
+    public void testCheckValidEmployeeId() {
+        doNothing().when(employeeFirebaseWorkflow).checkValidEmployeeId(anyString());
+        firebaseService.checkValidEmployeeId("e1");
+        verify(employeeFirebaseWorkflow, times(1)).checkValidEmployeeId("e1");
```
Comment (SarvagyaGupta) [inline src/test/java/com/careerfair/q/service/database/implementation/FirebaseServiceTest.java:143]: change "e1" to anyString()

File: src/test/java/com/careerfair/q/service/database/implementation/FirebaseServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,535 @@
...
+            firebaseService.getCompanyWithId("f10", "c1");
+            fail();
+        } catch (FirebaseException ex) {
+            assertEquals(ex.getMessage(),"No fair exists with id=f10");
+        }
+    }
+
+    @Test
+    public void testGetCompanyWithInvalidCompanyId() {
+        doThrow(new FirebaseException("The company with company id=c10 is not present " +
```
Comment (SarvagyaGupta) [inline src/test/java/com/careerfair/q/service/database/implementation/FirebaseServiceTest.java:238]: Just do something like "error". Should be enough

File: src/test/java/com/careerfair/q/service/database/implementation/FirebaseServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,535 @@
...
+                anyString());
+        doNothing().when(employeeFirebaseWorkflow).registerStudentToEmployee(anyString(),
+                anyString());
+        firebaseService.registerStudent("s1", "e1");
+        verify(studentFirebaseWorkflow, times(1)).registerEmployeeToStudent("s1", "e1");
+        verify(employeeFirebaseWorkflow, times(1)).registerStudentToEmployee("e1", "s1");
+    }
+
+    @Test
+    public void testRegisterStudentInvalidStudentId() {
```
Comment (SarvagyaGupta) [inline src/test/java/com/careerfair/q/service/database/implementation/FirebaseServiceTest.java:287]: Umm.... the method names are fucked

File: src/test/java/com/careerfair/q/service/database/implementation/FirebaseServiceTest.java (hunk 1 above)
Comment (amankagarwal) [reply src/test/java/com/careerfair/q/service/database/implementation/FirebaseServiceTest.java:143]: do you mean the one in verify?

**Pull request merged**